demo_serial/src/main/java/com/novoda/downloadmanager/demo/QueryForDownloadsAsyncTask.java

```diff
@@ -0,0 +1,57 @@
+package com.novoda.downloadmanager.demo;
```

Comment:
well they don't have anything shared. In order to avoid any confusion, I've just duplicated the old demo project. I've updated the package names here 0cae060